changes/unreleased/Changed-20240808-140420.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Changed
```

Comment:
Only features exposed to users must appear in the release note

Reply:
Sure, will remove it.